var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -171,8 +171,10 @@ class Vtk(CMakePackage):
     depends_on("utf8cpp", when="@9:")
     depends_on("gl2ps", when="@8.1:")
     depends_on("gl2ps@1.4.1:", when="@9:")
-    depends_on("proj@4", when="@8.2.0")
-    depends_on("proj@4:", when="@9:")
+    depends_on("proj@4", when="@:8.2.0")
+    depends_on("proj@4:7", when="@:8.2.0")
```

Comment:
Sorry, should have caught this earlier, but doesn't this now imply the `depends_on` above it?